pkg/apierrors/apierrors.go

```diff
@@ -63,262 +63,229 @@ func (e *APIError) WithMessage(msg string) *APIError {
 	}
 }
 
 func NewGithubRepositoryNotFoundError() *APIError {
 	return &APIError{
-		Code:    errCodeGithubRepositoryNotFound,
+		Code:    ErrCodeGithubRepositoryNotFound,
 		Message: "Github repository not found among accessible repositories.",
 	}
 }
 
 func NewProjectMemberNotFoundError() *APIError {
 	return &APIError{
-		Code:    errCodeProjectMemberNotFound,
+		Code:    ErrCodeProjectMemberNotFound,
 		Message: "Project member not found",
 	}
 }
 
 func NewProjectMemberUnprocessableError() *APIError {
 	return &APIError{
-		Code:    errCodeProjectMemberUnprocessable,
+		Code:    ErrCodeProjectMemberUnprocessable,
 		Message: "Project member unprocessable",
 	}
 }
 
 func NewReleaseUnprocessableError() *APIError {
 	return &APIError{
-		Code:    errCodeReleaseUnprocessable,
+		Code:    ErrCodeReleaseUnprocessable,
 		Message: "Release unprocessable",
 	}
 }
 
 func NewReleaseNotFoundError() *APIError {
 	return &APIError{
-		Code:    errCodeReleaseNotFound,
+		Code:    ErrCodeReleaseNotFound,
 		Message: "Release not found",
 	}
 }
 
 func NewReleaseDuplicateTitleError() *APIError {
 	return &APIError{
-		Code:    errCodeReleaseDuplicateTitle,
+		Code:    ErrCodeReleaseDuplicateTitle,
 		Message: "Release with the same title already exists",
 	}
 }
 
 func NewSlackIntegrationNotEnabledError() *APIError {
 	return &APIError{
-		Code:    errCodeSlackIntegrationNotEnabled,
+		Code:    ErrCodeSlackIntegrationNotEnabled,
 		Message: "Slack integration is not enabled.",
 	}
 }
 
 func NewGitTagNotFoundError() *APIError {
 	return &APIError{
-		Code:    errCodeGitTagNotFound,
+		Code:    ErrCodeGitTagNotFound,
 		Message: "Git tag not found",
 	}
 }
 
 func NewGithubReleaseNotFoundError() *APIError {
 	return &APIError{
-		Code:    errCodeGithubReleaseNotFound,
+		Code:    ErrCodeGithubReleaseNotFound,
 		Message: "Github release not found",
 	}
 }
 
-func IsErrorWithCode(err error, code string) bool {
-	var apiErr *APIError
-	if errors.As(err, &apiErr) {
-		return apiErr.Code == code
-	}
-
-	return false
-}
-
 func IsNotFoundError(err error) bool {
-	return IsErrorWithCode(err, errCodeUserNotFound) ||
-		IsErrorWithCode(err, errCodeProjectNotFound) ||
-		IsErrorWithCode(err, errCodeEnvironmentNotFound) ||
-		IsErrorWithCode(err, errCodeProjectInvitationNotFound) ||
-		IsErrorWithCode(err, errCodeGithubRepositoryNotFound) ||
-		IsErrorWithCode(err, errCodeGithubRepositoryNotConfiguredForProject) ||
-		IsErrorWithCode(err, errCodeGithubIntegrationNotEnabled) ||
-		IsErrorWithCode(err, errCodeProjectMemberNotFound) ||
-		IsErrorWithCode(err, errCodeGithubIntegrationNotEnabled) ||
-		IsErrorWithCode(err, errCodeGithubRepositoryInvalidURL) ||
-		IsErrorWithCode(err, errCodeReleaseNotFound) ||
-		IsErrorWithCode(err, errCodeGitTagNotFound) ||
-		IsErrorWithCode(err, errCodeGithubReleaseNotFound)
-}
-
-func IsUnprocessableModelError(err error) bool {
```

Comment:
`apierrors` now has only `IsXXXError(err error) bool` methods that are used in the service.